attributes/default.rb

```diff
@@ -21,7 +21,7 @@
 default['rocketchat']['group'] = 'rocketchat'
 default['rocketchat']['install_dir'] = '/srv/rocketchat'
 
-default['rocketchat']['url'] = 'https://rocket.chat/releases/latest/download'
```

Comment:
@jjasghar currently release is actually 0.56.0 so for one this would be several months out of date.  As far as i'm aware there is no reason to hang chef on ubuntu back or any other platform for that matter.

@maraaaa if there is some reason to hang it back please let me know and we'll get it resolved.